source/usage-examples/deleteOne.txt

```diff
@@ -3,3 +3,50 @@ Delete a Document
 =================
 
 .. default-domain:: mongodb
+
+You can delete a document in a collection using the ``DeleteOne()``
+method. 
+
+The following example specifies a query filter that matches documents in
+the ``movies`` collection with the value "Twilight" in the ``title`` field, and
+deletes the first document that matches. 
+
+.. include:: /includes/run-example-tip.rst
+
+.. literalinclude:: /includes/usage-examples/code-snippets/deleteOne.go
+   :start-after: begin deleteOne
+   :end-before: end deleteOne
+   :emphasize-lines: 2
+   :language: go
+   :dedent:
+
+Click here <TODO> to see a fully runnable example.
+
+Expected Result
+---------------
+
+After running the preceding code snippet, you should not be able to find
+the following document in the ``movies`` collection:
+
+.. code-block:: json
+
+    { "_id": { "$oid": "573a13bff29313caabd5e06b" }, 
+      ...
+      "title": "Twilight",
+      ...
+    }
+
+For an example on how to find a document, see our :doc:`Find
+One Usage Example </usage-examples/findOne>`.
+
+Additional Information
+----------------------
+
+For more information on deleting documents, specifying query filters,
+and handling potential errors, see our guide on <TODO:
+Deleting a Document>.
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+:go-api:`DeleteOne() <mongo#Collection.DeleteOne>`
```

Comment:
Similar comment as on https://github.com/mongodb/docs-golang/pull/5:
This link in on the staging server goes to [pkg.go.dev](http://pkg.go.dev) documentation for the `github.com/mongodb/mongo-go-driver` package, but the canonical package is `go.mongodb.org/mongo-driver`. They're technically the same, but it could be confusing if the reader doesn't realize that.

Please update to link to the docs at https://pkg.go.dev/go.mongodb.org/mongo-driver instead.